portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+import java.io.IOException;
+import java.util.*;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/add-comment")
+public class AddCommentServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String comment_text = getParameter(request, "comment-text", "");
+        String name = getParameter(request, "name", "");
+        String email = getParameter(request, "email", "");
+        long timestamp = System.currentTimeMillis();
+
+        Entity commentEntity = new Entity("Comment");
+        commentEntity.setProperty("comment-text", comment_text);
+        commentEntity.setProperty("name", name);
+        commentEntity.setProperty("email", email);
+        commentEntity.setProperty("timestamp", timestamp);
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        datastore.put(commentEntity);
+
+        response.sendRedirect("/index.html");
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+    private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
Can you check if this line exceeds 100 chars ? 
https://google.github.io/styleguide/javaguide.html#s4.4-column-limit